README.md

```diff
@@ -30,31 +38,11 @@ Or install it yourself as:
     $ gem install hydra-works
 -->
 
-## Access Controls
-We are using [Web ACL](http://www.w3.org/wiki/WebAccessControl) as implemented by [hydra-access](https://github.com/projecthydra/hydra-head/tree/master/hydra-access-controls) controls
-
-## Hydra Works Model
-
-PCDM Reference:  [Portland Common Data Model](https://wiki.duraspace.org/x/9IoOB)
-
-![Hydra Works Model Definition](https://docs.google.com/presentation/d/1d43d5o2d0x6NyI1tjjuJN6BDI9F9XYxZ1_9_EoZbAyE/edit#slide=id.p8)
-
-
 ## Usage
 
-Hydra-works provides three classes:
-```
-Hydra::Works::Collection - a collection aggregates collections and generic works
-Hydra::Works::GenericWork - a generic work aggregates generic works and generic files
-Hydra::Works::GenericFile - a generic file aggregates generic files and contains pcdm files
-```
-
-A `Hydra::PCDM::File` is a NonRDFSource &emdash; a bitstream.  You can use this to store content. A PCDM::File is contained by a PCDM::Object. A `File` has some attached technical metadata, but no descriptive metadata.  A `Hydra::PCDM::Object` contains files and other objects and may have descriptive metadata.  A `Hydra::PCDM::Collection` can contain other `Collection`s or `Object`s but not `File`s.  A `Collection` also may have descriptive metadata.
-
-Typically usage involves extending the behavior provided by this gem. In your application you can write something like this:
+Usage involves extending the behavior provided by this gem. In your application, you can create Hydra::Works-based models like so:
 
 ```ruby
-
 class Collection < ActiveFedora::Base
   include Hydra::PCDM::CollectionBehavior
```

Comment:
Remove this line. The Works::CollectionBehavior includes PCDM::Collection behavior for you.